code/modules/mob/living/living.dm

```diff
@@ -52,6 +52,10 @@
 
 /mob/living/ghostize(can_reenter_corpse = 1)
 	var/prev_client = client
+	if(isobj(loc))
+		var/obj/our_location = loc
+		if(length(our_location.client_mobs_in_contents))
+			our_location.client_mobs_in_contents -= src // if you jackhammer click this as an admeme you can cause runtimes without a length check
```

Comment:
I feel like this should be something that's handled in Logout or something like that. I can see multiple instances of this happening that ghostize might not always handle well.

Reply:
Any specific test case I can try where this might break? Bit nervous about putting it in loggout because it has a spawn

Reply:
Yeah----uhhhh, it probably shouldn't...

Reply:
works on logout now